src/pure-utils/pathToAction.js

```diff
@@ -10,7 +10,8 @@ export default (
   pathname: string,
   routesMap: RoutesMap,
   serializer?: QuerySerializer,
-  basename?: string | void = getOptions().basename
+  basename?: string | void = getOptions().basename,
+  strict?: boolean | void = getOptions().strict
```

Comment:
I tried `{ basename?: string, strict?: boolean } | void = getOptions` and many different alternatives with no success regarding Flow parsing.